src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +20,13 @@
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        return numbers.stream()
+                .map(word -> Stream.of(word.split(",")).collect(Collectors.toList()))
+                .flatMap(Collection::stream)
```

Comment:
```suggestion
                .flatMap(word -> Arrays.stream(word.split(",")))
```
Як на мене то зайві рухи. Ти спочатку стрімиш String, потім стрімиш List<String> , а потім знову String.

Reply:
видалив зайві кроки